main.go

```diff
@@ -44,8 +47,9 @@ import (
 )
 
 var (
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme     = runtime.NewScheme()
+	setupLog   = ctrl.Log.WithName("setup")
+	logOptions = logs.NewOptions()
```

Comment:
Please move this to the second var block, that's what upstream is doing in newer versions of the file.

Reply:
This is not big enough to block on, resolving for now.